README.md

```diff
@@ -1,3 +1,17 @@
+
+# setup
```

Comment:
Would you mind moving this in a sub-section? Something like:

```
# strava-cli

Command line interface for [Strava](http://www.strava.com). It allows to:
...

## Setup

To use strava-cli you will need python3 and these dependencies:
...

```

I think it will be more consistent with the README structure. Thanks!

Reply:
I've made this change, thanks!

Please, though, I wasn't sure how far down to put the "Setup" section, so feel free to move it or tell me where to move it to.